packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -80,6 +80,14 @@ export class TablePageObject<T extends TableRecord> {
         );
     }
 
+    public getHeaderTextContent(columnIndex: number): string {
```

Comment:
As mentioned in the PR description, this function was copied by the `sl-table` page object.